book/src/powdr_crate.md

```diff
@@ -0,0 +1,33 @@
+# Using powdr as a library
+
+Besides the CLI, powdr can also be used as a Rust library.
+The [powdr crate](https://github.com/powdr-labs/powdr/tree/main/powdr)
+exposes internal crates and data structures needed to compile code,
+generate witnesses, and compute proofs.
+
+Add `powdr` to your crate's dependencies:
+
+```toml
+[dependencies]
+powdr = { git = "https://github.com/powdr-labs/powdr", branch = "main" }
+```
+
+The following Rust code has the same workflow as our previous "Hello World"
+example.  The full project can be found
+[here](https://github.com/powdr-labs/powdr-hello-world), and as an example in
```

Comment:
Is there a difference between this and `powdr/examples/powdr_crate_usage.rs`?

Reply:
no but I wanted to give an example with a full crate also with the import, so people can potentially fork it and go from there